src/index.js

```diff
@@ -670,7 +671,18 @@ const members = {
       return this.getAttribute('name');
     },
     set(name) {
-      return this.setAttribute('name', name);
+      const ret = this.__setAttribute('name', name);
+      if (!isSlotNode(this)) {
```

Comment:
I would also check the previous value if possible to make sure it's different before doing any unnecessary processing.